Makefile

```diff
@@ -0,0 +1,42 @@
+FMT_OPTS = -co -XTypeApplications -o -XImportQualifiedPost
+FIND_OPTS = src test -type f -name '*.hs'
+
+build:
+	cabal build
+
+test:
+	cabal test
+
+# Dogfood the dev version
+analyze:
+	cabal run fossa -- analyze --output --debug --filter 'cabal@./'
+
+# Copy the built binary into the local root
+fossa:
+	cp $(shell cabal list-bin fossa) ./fossa
+
+install-local: fossa
+
+# Run analysis on the sandbox directory
+sandbox: fossa
+	./fossa analyze --output --debug --record ./sandbox > fossa.json
```

Comment:
Like above, I'd remove this unless other folks are already using `./sandbox` as a convention.